src/main/resources/public/js/realmapps.js

```diff
@@ -409,7 +409,8 @@ angular.module('aac.controllers.realmapps', [])
         var slug = $stateParams.realmId;
         var clientId = $stateParams.clientId;
 
-        $scope.isRealmAdmin = $rootScope.user.authorities.some(a => (a.realm === slug && a.role === 'ROLE_ADMIN'));
+        var isSystemRealmAdmin = $rootScope.user.authorities.some(a => (a.realm === "system" && a.role === 'ROLE_ADMIN'));
```

Comment:
1. rename to `isAdmin` and set in `$scope`
2. admin has `authority === 'ROLE_ADMIN'`